src/sagas/projects.js

```diff
@@ -112,6 +114,23 @@ export function* projectExported({payload: {exportType}}) {
   }
 }
 
+export function* archiveProject({payload: {projectKey}}) {
+  const state = yield select();
+  const currentProjectKey = getCurrentProjectKey(state);
+  if (projectKey === currentProjectKey) {
+    yield put(createNewProject());
```

Comment:
Would it be simpler to just not show the archive button on the current project? Or would that be confusing from a UI standpoint?

Reply:
Would be a bit easier to implement. I will try it out to not show the archive button on the current project